tools/clang/test/HLSLFileCheck/info/dump_dependency/dump_dependency.hlsl

```diff
@@ -0,0 +1,20 @@
+// RUN: %dxc -E main -T ps_6_0 -M %s | FileCheck %s
+// RUN: %dxc -E main -T ps_6_0 -MF%s.deps %s && cat %s.deps | FileCheck %s
```

Comment:
How does this succeed on Windows in TAEF?  These commands are not run in a shell (see FileCheckerTest.cpp), so `&&` won't separate commands, and `cat` wouldn't be understood.  Additionally, -MF and -MD are not handled by the DxCompiler API, so this would have to actually be run through dxc.exe command line interface in order to test this.

For command line tests on Windows targets, we use `utils\hct\hcttestcmds.cmd`.

Reply:
Of course removing the options would simplify things, since it would resolve the test issue I raised without requiring you to implement separate tests in hcttestcmds.cmd. 😃

Reply:
I moved this unittests to `utils\hct\hcttestcmds.cmd`. Thank you for clarifying it!